### PR TITLE
Add database options to aws/azure/universal scalardl modules

### PR DESCRIPTION
**Pull request by ymorimo**
https://scalar-labs.atlassian.net/browse/DLT-7415

Add `database`, `database_contact_points` and `database_contact_port` variables to Terraform modules.

## To-Dos

The latest `scalar-ledger` image and `scalardl-schema-loader` image don't support DynamoDB yet. Their versions should be updated to work with DynamoDB.

### CODE REVIEW

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Comment by feeblefakie** [comment]
@tei-k Please take a look!

File: modules/universal/scalardl/main.tf [hunk 1]
```diff
@@ -189,7 +189,13 @@ resource "null_resource" "scalardl_schema" {
 
   provisioner "remote-exec" {
     inline = [
-      "docker run --rm ${var.schema_loader_image} --cassandra -h ${var.database_contact_points} -u ${var.database_username} -p ${var.database_password} -n NetworkTopologyStrategy -R ${var.cassandra_replication_factor}"
+      format(
+        "%s %s",
+        "docker run --rm ${var.schema_loader_image} -h ${var.database_contact_points} -u ${var.database_username} -p ${var.database_password}",
+        var.database == "cassandra" ? "--cassandra -P ${var.database_contact_port} -n NetworkTopologyStrategy -R ${var.cassandra_replication_factor}" :
+        var.database == "dynamo" ? "--dynamo --region ${var.database_contact_points}" :
```
Comment (tei-k) [inline modules/universal/scalardl/main.tf:196]: @ymorimo 

I'm not sure how it works with `--region ${var.database_contact_points}`. 💦

**Review by tei-k — APPROVED** [review]
Overall looking good. Left one qustion.

File: modules/universal/scalardl/main.tf (hunk 1 above)
Comment (ymorimo) [reply modules/universal/scalardl/main.tf:196]: Good point. When DynamoDB is used, the region name should be specified in the `contact_points` variable.
https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-dynamodb.md#configure-scalar-db
This is also added in the `database_contact_points` variable's description. https://github.com/scalar-labs/scalar-terraform/pull/237/files#diff-c29567323bde55c7871f3e1b1fd9242c4f268d7d82129d0eb5fce499a78319d4R18

**Review by yito88 — APPROVED** [review]
LGTM!

**Pull request merged**